### PR TITLE
add description and example for multiple columns in orderBy section

### PR DESCRIPTION
**Pull request by HurSungYun**
add documentation for

https://github.com/tgriesser/knex/pull/2881

### CODE REVIEW

File: sections/builder.js [hunk 1]
```diff
@@ -1035,13 +1035,39 @@ export default [
     type: "method",
     method: "orderBy",
     example: ".orderBy(column, [direction])",
-    description: "Adds an order by clause to the query.",
+    description: "Adds an order by clause to the query. column can be string or list of string and object.",
```
Comment (kibertoad) [inline sections/builder.js]: "column" probably should be "Column"?
"can be string or list of string and object" -> do I understand correctly that what you mean is "can be string, list of strings or an object"? Or you mean "can be string, list of string or list of objects"?

Comment (HurSungYun) [reply sections/builder.js]:
```
"column" probably should be "Column"?
```
`column` is a parameter for the method. Nevertheless, does it needs to be `Column`? 

```
"can be string or list of string and object" -> do I understand correctly that what you mean is "can be string, list of strings or an object"? Or you mean "can be string, list of string or list of objects"?
```
The latter would be right.
I will change this more clearly. Please understand me that I'm not English native :)

Comment (kibertoad) [reply sections/builder.js]: 1) Right, then probably you want to enclose it in backticks (``) for visual highlight.
2) Np, that's a very minor thing.

Thanks a lot for your contribution ;)

Comment (HurSungYun) [reply sections/builder.js]: This might be my first `real world` open source contribution.

and I thank you for guiding me very kindly.

👍

**Comment by kibertoad** [comment]
@HurSungYun Thanks! We'll merge it before the release with a new feature.

**Comment by HurSungYun** [comment]
https://github.com/tgriesser/knex/pull/2881

FYI

It has merged.

**Comment by elhigu** [comment]
This was missing a change for method signature types in orderBy header ".orderBy(column, [direction])",

**Pull request merged**